.gitignore

```diff
@@ -0,0 +1,3 @@
+
+__pycache__/
+/venv/
```

Comment:
Needs a new line at EOF (and has an extra line at the top)

Reply:
👍🏻